src/react-query/queryClient.ts

```diff
@@ -13,8 +14,25 @@ export const queryClient = new QueryClient({
   },
 });
 
+
+const storage = new MMKV();
+
+const clientStorage = {
+  setItem: (key, value) => {
+    storage.set(key, value);
+  },
+  getItem: (key) => {
+    const value = storage.getString(key);
+    return value === undefined ? '' : value;
+  },
+  removeItem: (key) => {
+    storage.delete(key);
+  },
+};
+
 const asyncStoragePersister = createAsyncStoragePersister({
-  key: 'rainbow.react-query',
+  key: 'rainbow.react-query.mmkv',
```

Comment:
Does changing this key have any side effects?

Reply:
side effect being the previous favorites are not overwritten when the app launch and our persister configures itself